server/api/index.ts

```diff
@@ -1,11 +1,21 @@
+import { cleanEnv } from 'envalid';
 import { AsyncRouter } from 'express-async-router';
 
+import devRouter from 'server/api/dev';
+
 import authRouter from './auth';
 import userRouter from './users';
 
 const apiRouter = AsyncRouter();
 
+const env = cleanEnv(process.env, {});
+
 apiRouter.use('/users', userRouter);
 apiRouter.use('/auth', authRouter);
 
+if (!env.isProd) {
```

Comment:
i think it's safer to use `env.isDev` here (that way it's _only_ set is NODE_ENV is development, whereas if it's unset for whatever reason, or set to something similar to but not exactly "production", this will still not apply)